databuilder/models/application.py

```diff
@@ -96,7 +97,7 @@ def create_nodes(self):
             label=Application.APPLICATION_LABEL,
             node_attributes={
                 Application.APPLICATION_URL_NAME: self.application_url,
-                Application.APPLICATION_NAME: Application.APPLICATION_TYPE,
+                Application.APPLICATION_NAME: self.app_name,
                 Application.APPLICATION_DESCRIPTION: application_description,
                 Application.APPLICATION_ID: application_id
```

Comment:
This `APPLICATION_ID` attribute is not supported in the Neptune output, it collides with the `id` of the node and simply gets dropped. In the corresponding PR to the metadatalibrary I just ignored it. It would be a bit less sketchy to rename the field to... `dag_id/task_id` is not a very meaningful thing to call `id` anyway. But not sure what a better name would be.